### ISSUE TITLE
Fixed input image shape

### ISSUE THREAD

**Issue by Christinepan881** [facebookresearch/Mask2Former#67]
Hi, thanks for the awesome work.

Does this model handle the various input shape or just fixed size of images?
In `Mask2Former/mask2former/modeling/pixel_decoder/msdeformattn.py` , the forward part of `class MSDeformAttnTransformerEncoderOnly(nn.Module)`: 
`masks = [torch.zeros((x.size(0), x.size(2), x.size(3)), device=x.device, dtype=torch.bool) for x in srcs]` 
The masks are all zeros, so it assumes the input image size is fixed?

Thanks!

**Comment by bowenc0221** [comment]
The model can handle arbitrary input size.
